scrapy/templates/project/module/middlewares.py.tmpl

```diff
@@ -7,6 +7,9 @@
 
 from scrapy import signals
 
+# useful for handling different item types with a single interface
+from scrapy.utils.item import ItemAdapter
+
```

Comment:
@kmike Added these lines to both templates, let me know if you had something else in mind

Reply:
I was thinking we can have it in the example code below as well- something like

```py
adapter =ItemAdapter(item)
# ... processing code should work with adapter, not with an item;
# item is going to be modified in-place
return item 
```

But unsure now, as it starts to feel heavy.